src/Auxiliary/Aux_Check_Parameter.cpp

```diff
@@ -716,7 +716,17 @@ void Aux_Check_Parameter()
 #     elif ( NUC_TABLE_MODE != NUC_TABLE_MODE_TEMP  &&  NUC_TABLE_MODE != NUC_TABLE_MODE_ENGY )
 #        error : ERROR : unsupported nuclear EoS table type (NUC_TABLE_MODE_TEMP/NUC_TABLE_MODE_ENGY) !!
```

Comment:
is `NUC_TABLE_MODE ` works with the `LUT` only?

Reply:
Yes. I also consider to remove `NUC_TABLE_MODE` if we will never use energy-based tables.